plugins/poll/assets/javascripts/discourse/components/poll.gjs

```diff
@@ -452,6 +458,17 @@ export default class PollComponent extends Component {
     return htmlSafe(I18n.t("poll.average_rating", { average }));
   }
 
+  get availableDisplayMode() {
```

Comment:
I'm a little bit confused by this function. It's named in singular, and it seems to only ever return a single element, but the return type is an array.

If this potentially needs to return multiple elements, maybe we can rename it to `#availableDisplayModes` (plural). If not, maybe we can make it return a string?

Reply:
You are right! Here is what I wrote while thinking, now I changed it to a string.